contracts/oracles/Oracle.sol

```diff
@@ -23,10 +23,10 @@ contract Oracle is Ownable {
     uint8 public resultIndex = invalidResultIndex;
     uint16 public version;
     bytes32[10] public eventName;
-    bytes32[10] public eventResultNames;
+    bytes32[11] public eventResultNames;
     address public eventAddress;
     uint256 public consensusThreshold;
-    ResultBalance[10] internal resultBalances;
+    ResultBalance[11] internal resultBalances;
 
```

Comment:
I've proposed this before. Can we use constants on top of this page to represent 10 and 11? Is it doable? With a name it's less likely for us to mistake A as B, and don't need to update multiple places upon change.

Reply:
I need to see if var names (in this case, an array capacity) can take constants that were defined. And if so, I would have to make a `BaseContract` which `TopicEvent` and `Oracle` inherit from.

Reply:
Let me test this out with the consts used in var names.

Reply:
Ya not possible to have constants as the param in array capacities. I get this error:
```
/Users/deric/coding/bodhi-core/contracts/events/TopicEvent.sol:45:13: TypeError: Invalid array length, expected integer literal.
    bytes32[NAME_ARR_LEN] public name;
```

Reply:
But that being said, I am going to add a base contract class to have reusable elements between TopicEvent and Oracle 👍